monkestation/code/modules/blood_for_the_blood_gods/slasher/abilities/envelope_darkness.dm

```diff
@@ -50,8 +50,12 @@
  * Puts owner in the phased mob holder here.
  */
 /datum/action/cooldown/slasher/envelope_darkness/proc/do_jaunt(mob/living/jaunter)
-	ADD_TRAIT(jaunter, TRAIT_NO_TRANSFORM, REF(src))
+	ADD_TRAIT(jaunter, TRAIT_NO_TRANSFORM,  REF(src))
 	var/obj/effect/dummy/phased_mob/slasher_jaunt/holder = enter_jaunt(jaunter)
+	var/obj/cuffs = jaunter.get_item_by_slot(ITEM_SLOT_HANDCUFFED)
+	jaunter.visible_message("You see [jaunter] fade away into nothingness!")
+	if(HAS_TRAIT(jaunter, TRAIT_RESTRAINED))
+		qdel(cuffs)
```

Comment:
Don't bother with all this.
Just do `jaunter.uncuff()` and let the proc do it's thing